src/app/components/common/navigation/navigation.component.scss

```diff
@@ -23,4 +23,9 @@
   .nav-section-divider {
     margin: 0 -24px;
   }
+
+  .sidenav-link:focus-visible {
+    box-shadow: 0 0 0 1px var(--alt-fg2) inset;
```

Comment:
Should be a bit more subtle. Try `0.5px` here and all other places like this.

Reply:
Unfortunately 0.5px does not work correctly, because A pixel is the smallest unit value to render something.

Reply:
Please review this
https://stackoverflow.com/a/45522942/6664821